docs/hub/spaces-add-to-arxiv.md

```diff
@@ -1,30 +1,32 @@
 # How to Add a Space to ArXiv
 
 Demos on Hugging Face Spaces allow a wide audience to try out state-of-the-art machine
-learning research without writing any code. Hugging Face and ArXiv have collaborated 
+learning research without writing any code. [Hugging Face and ArXiv have collaborated](https://huggingface.co/blog/arxiv) 
 to embed these demos directly along side papers on ArXiv!
 
-Thanks to this integration, users can now find the most popular demos for a paper on its arXiv abstract page. For example, if you want to try out demos of the LayoutLM document classification model, you can go to [the LayoutLM paper's arXiv page](https://arxiv.org/abs/1912.13318), and navigate to the demo tab. You will see open-source demos built by the machine learning community for this model.
+Thanks to this integration, users can now find the most popular demos for a paper on its arXiv abstract page. For example, if you want to try out demos of the LayoutLM document classification model, you can go to [the LayoutLM paper's arXiv page](https://arxiv.org/abs/1912.13318), and navigate to the demo tab. You will see open-source demos built by the machine learning community for this model, which you can try out immediately in your browser:
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/layout-lm-space-arxiv.gif)
```

Comment:
Was this gif deleted? https://huggingface.co/datasets/huggingface/documentation-images/commit/27b5026dd00e2c6571651b201d59611608772f71

Reply:
Haha you're too quick! Just fixed, the GIF was uploaded in the wrong location

Reply:
Nice gif!